tests/unit_tests/docmaps/v2/codecs/elife_manuscript_test.py

```diff
@@ -131,6 +132,29 @@ def test_should_populate_elife_manuscript_part_of_section(self):
             )
         }
 
+    def test_should_have_same_manuscript_volume_for_each_version_of_manuscripts(self):
```

Comment:
Maybe we could also test that in this case it is the volume identifier according to the publication date of the first version? (otherwise it could be the last version and pass the test)

Reply:
Actually you are right. If there is no 'rp_publication_timestamp' we are not even providing manuscript-published step. I will fix this.